circleci/images/exttester/Dockerfile

```diff
@@ -1,16 +1,140 @@
-FROM buildpack-deps@sha256:19a482aad09128bd0c448373d933aa0f108dc8eb9be2317dca3797c646ce4b57
+# postgres doesn't ship their isolation tester in the server-dev images
+# we use this builder to build the isolation tester from a specific version
+FROM buildpack-deps:stretch AS dev-tools-builder
 
+RUN apt-get update \
+# install dependencies
+ && apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    debian-archive-keyring \
+    gcc \
+    gnupg \
+    gosu \
+    make \
+    perl \
+    autoconf \
+    build-essential \
+    libcurl3 \
+    libcurl4-openssl-dev \
+    libicu-dev \
+    libreadline-dev \
+    libselinux1-dev \
+    libxslt-dev \
+    libssl-dev \
+    locales \
+# clear apt cache
+ && rm -rf /var/lib/apt/lists/*
+
+ARG PG_VERSION
 ARG PG_MAJOR
-ENV PG_MAJOR $PG_MAJOR
-ENV PATH=/usr/lib/postgresql/$PG_MAJOR/bin/:$PATH
 
-COPY files /tmp
+WORKDIR /build/
+RUN curl -fLO "http://ftp.postgresql.org/pub/source/v${PG_VERSION}/postgresql-${PG_VERSION}.tar.bz2"
+
+RUN tar jxf "postgresql-${PG_VERSION}.tar.bz2"
+WORKDIR /build/postgresql-${PG_VERSION}/build
+RUN ../configure --prefix /usr/lib/postgresql/${PG_MAJOR}/
+RUN make -sj8
+
+RUN ls /build/postgresql-${PG_VERSION}/build/
```

Comment:
yeah, the ls was debugging, I can remove them. I don't think they are required anymore.

In later iterations I have been `docker run`ing the intermediate layers here to inspect them instead of `ls`ing in the build.